README.md

```diff
@@ -827,10 +827,11 @@ created in a constructor but in create_thread function. Program will
 terminate if there is no resources. Alternatively, the function
 could return false instead. By default 512 words will be allocated
 for the stack. This would be 2kB on ARM. Standard C++ interface does not let
-define the stack size. So, this code has to be modified if application 
-requires more. Note, that the change will apply to all threads. The 2kB is 
-required when futures are used. Without futures, I had the system running 
-with 1kB only. 
+define the stack size. It is possible to configure the default stack size (in
+words) by setting the macro `configDEFAULT_STD_THREAD_STACK_SIZE` in your
+`FreeRTOSConfig.h` file. Note, that the change will apply to all threads. The
+2kB is required when futures are used. Without futures, I had the system
+running with 1kB only. 
```

Comment:
Thanks for updating README. Once at it, can you update the code snippet below to reflect the fact that the stack size is not fixed now.

Line 853, here: https://github.com/grygorek/FreeRTOS_cpp11/pull/40/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R853

Reply:
Ah, I missed that one :) fixed now